project/src/main/webapp/index.html

```diff
@@ -2,17 +2,26 @@
 <html>
```

Comment:
This file looks like it's out of date. Try `git pull origin master` to update it and resolve any merge conflicts (let me know if you have trouble with this)